qml/ConnectionMenu.qml

```diff
@@ -197,9 +203,29 @@ Item {
                     text: "1234"
                     Layout.columnSpan:  2
                     Layout.fillWidth: true
-                    onEditingFinished: {
-                        connect(AbstractLinkNamespace.Udp, udpIp.text, text)
+                }
+            }
+
+            Button {
+                text: "Connect"
+                Layout.fillWidth: true
+                Layout.columnSpan: 5
+                onClicked: {
```

Comment:
maybe this funcitons should go to c++ ?

Reply:
There is a C++ function in the end to do the connection, this function only gets the necessary information for each QML control.